d2/src/main/java/com/linkedin/d2/balancer/properties/ServiceProperties.java

```diff
@@ -140,9 +140,12 @@ public String getClusterName()
     return _clusterName;
   }
 
+  /**
+   * @return Prioritized {@link com.linkedin.d2.balancer.strategies.LoadBalancerStrategy} list.
```

Comment:
Yes, list is still necessary since the core problem will always exist. Server can define a strategy that they want, but clients may not be on latest version with the strategy yet. Especially now since we will be doing another migration for this strategy.